scheduler/src/test/java/gov/nasa/jpl/aerie/scheduler/StepSimulationTest.java

```diff
@@ -0,0 +1,64 @@
+package gov.nasa.jpl.aerie.scheduler;
+
+import gov.nasa.jpl.aerie.merlin.driver.SerializedActivity;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import org.apache.commons.lang3.tuple.Triple;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.SECOND;
+
+public class StepSimulationTest {
+
+  @Test
+  public void resetEveryTime() {
+    manyacts(true);
+  }
+  @Test
+  public void dontreset() {
+    manyacts(false);
+  }
+
+  public void manyacts(boolean reset){
+    var cur = Duration.ZERO;
+    final var step = Duration.of(3,SECOND);
+    // ten days
+    final var durSim = Duration.of(864000, SECOND);
```

Comment:
We can internalize the comment, e.g. as:
```suggestion
    final var DAY = Duration.HOUR.times(24);
    final var durSim = DAY.times(10);
```

Reply:
The way activities are generated has changed.